common/djangoapps/enrollment/views.py

```diff
@@ -712,10 +712,12 @@ class CourseEnrollmentsApiListView(DeveloperErrorViewMixin, ListAPIView):
     """
         **Use Cases**
 
-            Get a list of all course enrollments given a course ID or list of usernames.
+            Get a list of all course enrollments, optionally filtered by a course ID or list of usernames.
```

Comment:
nit: "optionally filtered by a course ID *and/or* list of usernames"

I know.. programmatically "and/or" is the same as "or", but to humans, I think this is clearer.

Reply:
👍